py/STREAM/Settings/TransportSettings.py

```diff
@@ -1,5 +1,5 @@
 import numpy as np
-#from .. DREAMException import DREAMException
+# from .. DREAMException import DREAMException
```

Comment:
consider erasing if it shall remain as a comment.

Reply:
Done